core/domain/html_cleaner.py

```diff
@@ -136,11 +136,12 @@ def get_rte_components(html_string):
             customization_args = {}
             for attr in oppia_custom_tag_attrs[tag_name]:
                 # Unescape special HTML characters such as '&quot;'.
-                attr_val = parser.unescape(component_tag[attr])
-                # Adds escapes so that things like '\frac' aren't
-                # interpreted as special characters.
-                attr_val = attr_val.encode('unicode_escape')
-                customization_args[attr] = json.loads(attr_val)
+                if attr in str(component_tag):
```

Comment:
Why wouldn't it be?

/cc @bansalnitish

Reply:
Oh my bad. I will remove this.